Libplanet/Blocks/Block.cs

```diff
@@ -168,6 +167,7 @@ private Block(RawBlock rb)
         /// A cancellation token used to propagate notification that this
         /// operation should be canceled.</param>
         /// <returns>A <see cref="Block{T}"/> that mined.</returns>
+
```

Comment:
This blank line should be removed ([SA1506](http://stylecop.soyuz5.com/SA1506.html)).

```suggestion

```